libraries/lib-cloud-audiocom/ServiceConfig.cpp

```diff
@@ -55,11 +55,6 @@ std::chrono::milliseconds ServiceConfig::GetProgressCallbackTimeout() const
    return std::chrono::seconds(3);
 }
 
-AudioFormat ServiceConfig::GetPreferredAudioFormat() const
```

Comment:
I strongly disagree with this change. You lift responsibility from the business logic into the UI.